src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -76,7 +76,7 @@
                   :header-cell {:padding "0.5rem 0 0.5rem 1rem"}
                   :body-cell {:padding "0.3rem 0 0.3rem 1rem"}}}
          :toolbar ;; FIXME: magic numbers below:
-         {:items (constantly
+         {:get-items (constantly
```

Comment:
This file also makes use of a ':selected-items'. Is that something that should be changed to ':get-selected-items'?